lib/glue/tracker.rb

```diff
@@ -33,6 +33,18 @@ def report finding
     @findings << finding
   end
 
+  def get_worst_finding
+    worst = nil
+    @findings.each do |finding|
+      if !worst
+        worst = finding
+      elsif finding.severity > worst.severity
```

Comment:
We may want to review this as well, as for some tasks (tools), this will be doing string comparisons instead of the numeric comparisons you want (e.g. "Medium" > "High").